testing/upload_validation.ts

```diff
@@ -498,7 +541,7 @@ const paramDefValidator = zodCompleteObject<ParamDef<any>>({
         message: 'Object parameters are not currently supported.',
       },
     ),
-  description: z.string().max(Limits.BuildingBlockDescription),
+  description: descriptionTokens,
```

Comment:
Similar throughout, have to do a union of string and this new type to support down-version SDK clients.

If you want to support that current SDK versions are passing tokens,  can add that as an add-on check within `packMetadataSchemaBySdkVersion` in this file.

Reply:
Done. Added tests as well.